cmd/autoheal/healer.go

```diff
@@ -132,6 +137,7 @@ func (h *Healer) Run(stopCh <-chan struct{}) error {
 	// Start the workers:
 	go wait.Until(h.runRulesWorker, time.Second, stopCh)
 	go wait.Until(h.runAlertsWorker, time.Second, stopCh)
+	go wait.Until(h.RunActiveJobsWorker, time.Minute, stopCh)
```

Comment:
TODO: make this configurable.
This is 1 minute for debug purposes. default will be 5 minutes